### PR TITLE
Feat/panel toggle

### PR DESCRIPTION
**Pull request by nikolabulatovic**

### CODE REVIEW

File: src/app/core/workbox/workbox.service.ts [hunk 1]
```diff
@@ -37,6 +37,7 @@ export class WorkboxService {
 
     public openTab(tab, persistToRecentApps: boolean = true) {
 
+        debugger;
```
Comment (ivanbatic) [inline src/app/core/workbox/workbox.service.ts]: remove

File: src/app/layout/status-bar/status-bar.component.ts [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
 @Component({
     selector: "ct-status-bar",
     styleUrls: ["./status-bar.component.scss"],
     template: `
 
-        <!--Status-->
-        <span class="status-item">
+        <!--Status & panel toggle button-->
+        <span class="status-item status-buttons">
+            <button class="panel-toggle btn btn-sm" [ngClass]="!layoutService.sidebarExpanded ? 'active' : ''"
```
Comment (ivanbatic) [inline src/app/layout/status-bar/status-bar.component.ts]: [class.active]="!layoutService.sidebarExpanded"

File: src/app/core/layout/layout.service.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import {Injectable} from "@angular/core";
+
+@Injectable()
+export class LayoutService {
+    sidebarExpanded: boolean = true;
```
Comment (ivanbatic) [inline src/app/core/layout/layout.service.ts]: No need for ": boolean", type is inferred from the assignment

File: src/app/layout/status-bar/status-bar.component.scss [hunk 4]
```diff
@@ -9,6 +9,14 @@
     height: 2em;
     line-height: $layout-stripe-width - 6;
 
+    .panel-toggle {
```
Comment (ivanbatic) [inline src/app/layout/status-bar/status-bar.component.scss]: Not needed (except for font-size) if put the button.panel-toggle inside a div.btn-group.

File: src/app/core/layout/layout.service.ts [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+
+    toggleSidebar() {
+        this.sidebarExpanded = !this.sidebarExpanded;
```
Comment (ivanbatic) [inline src/app/core/layout/layout.service.ts]: Would be cool if value gets preserved in user preferences

**Comment by mayacoda** [comment]
Make LayoutService public in LayoutComponent: 

```
ERROR in ng:///home/travis/build/rabix/cottontail-frontend/src/app/core/layout/layout.component.ts.LayoutComponent.html (9,39): Property 'layoutService' is private and only accessible within class 'LayoutComponent'.
```

**Pull request merged**